obal/data/roles/fetch_sources/tasks/main.yml

```diff
@@ -108,6 +108,22 @@
         - '%global mainver'
       when: version is defined
 
+    - name: 'Bump prerelease in specfile'
+      replace:
+        path: "{{ spec_file_path }}"
+        regexp: '^(?P<prefix>{{ item }}\s+).*$'
+        replace: '\g<prefix>{{ prerelease }}'
+      with_items:
+        - '%global prerelease'
+      when: prerelease is defined
+
+    - name: 'Remove prerelease in specfile'
+      lineinfile:
+        path: "{{ spec_file_path }}"
+        regexp: '^%global prerelease'
+        state: absent
+      when: prerelease is not defined
```

Comment:
At first, glance, this seems like odd behavior in a `fetch_sources` role.

Reply:
Technically, I agree, but we have all the modifications in that role at the moment.

I can go and try untangle this, but I fear it's a chunk of work.